mm2src/mm2_net/src/network_event.rs

```diff
@@ -30,18 +30,18 @@ impl EventBehaviour for NetworkEvent {
         loop {
             let p2p_cmd_tx = p2p_ctx.cmd_tx.lock().clone();
 
-            let peers_info = atomicdex::get_peers_info(p2p_cmd_tx.clone()).await;
+            let seeds_info = atomicdex::get_directly_connected_peers(p2p_cmd_tx.clone()).await;
             let gossip_mesh = atomicdex::get_gossip_mesh(p2p_cmd_tx.clone()).await;
             let gossip_peer_topics = atomicdex::get_gossip_peer_topics(p2p_cmd_tx.clone()).await;
             let gossip_topic_peers = atomicdex::get_gossip_topic_peers(p2p_cmd_tx.clone()).await;
             let relay_mesh = atomicdex::get_relay_mesh(p2p_cmd_tx).await;
 
             let event_data = json!({
-                "peers_info": peers_info,
                 "gossip_mesh": gossip_mesh,
                 "gossip_peer_topics": gossip_peer_topics,
                 "gossip_topic_peers": gossip_topic_peers,
                 "relay_mesh": relay_mesh,
+                "seeds_info": seeds_info,
```

Comment:
Should this still be called `seeds_info`?

Reply:
Nope, updated.